crates/rooch-framework/sources/account_authentication.move

```diff
@@ -37,6 +37,12 @@ module rooch_framework::account_authentication{
       }
    }
 
+   #[private_generics(ValidatorType)]
+   /// Entry function of rotate_authentication_key. Do we need to limit the scope using private_generics?
+   public entry fun rotate_authentication_key_entry<ValidatorType>(ctx: &mut StorageContext, account: &signer, new_auth_key: vector<u8>) {
+      rotate_authentication_key<ValidatorType>(ctx, account, new_auth_key);
+   }
```

Comment:
@steelgeek091 This call should be failed when compiled. The private generics Type can not pass through.

Reply:
Record an issue #578